File: BE/health-service/src/main/java/com/s206/health/bodyinfo/dto/response/MonthlyAverageBodyInfoResponse.java
package com.s206.health.bodyinfo.dto.response;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class MonthlyAverageBodyInfoResponse {

	private int monthIndex; // 1~7
	private BigDecimal avgWeight;
	private BigDecimal avgMuscleMass;
	private BigDecimal avgBodyFat;

}

File: FE/mobile/src/main/java/com/example/diaviseo/network/body/dto/res/MonthlyAverageBodyInfoResponse.kt
package com.example.diaviseo.network.body.dto.res

data class MonthlyAverageBodyInfoResponse(
    val monthIndex: Int,  // 1~7, 필수 값이므로 non-nullable
    val avgWeight: Double,
    val avgMuscleMass: Double,
    val avgBodyFat: Double,
    val avgHeight: Double
)
